tools/deb_to_rb.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+set -e
+
+CREW_PREFIX=${CREW_PREFIX:-/usr/local}
+CREW_LIB_PREFIX=${CREW_PREFIX}/lib
+PKG_PREFIX=${CREW_LIB_PREFIX}/crew/packages
+GREEN='\e[1;32m';
+RED='\e[1;31m';
+BLUE='\e[1;34m';
+YELLOW='\e[1;33m';
+RESET='\e[0m'
+
+rm -rf ${CREW_PREFIX}/tmp/deb && cd ${CREW_PREFIX}/tmp/
+
+mkdir -p deb && cd ./deb
+
+echo -e "${YELLOW}Downloading deb...${RESET}"
+curl '-#' ${1} -o deb
+
+ar x deb
+
+
+tar xf control.tar.*
+  
+NAME=$(grep "Package: " ./control | sed -e 's/Package: //')
+PKGNAME=$(echo ${name} | sed -e 's/-/_/g')
+VER=$(grep "Version: " ./control | sed -e 's/Version: //' | sed -e 's/-/_/g')
+INFO=$(grep "Description: " ./control | sed -e 's/Description: //' | sed -e 's/-/_/g')
+SHA256SUM=$(sha256sum ./deb | cut -d ' ' -f1)
+
+OUT=${2}/${PKGNAME}.rb
+  
+cat <<EOF> ${OUT}
+require 'package'
+class ${PKGNAME^} < Package
+  description '${INFO}'
+  homepage ''
+  compatibility 'all'
+  @_ver = '${VER}'
+  version @_ver
+  
+  case ARCH
+  when 'x86_64'
+    source_url '${1}'
+    source_sha256 '${SHA256SUM}'
+EOF
+DEPS=($( sed '/Pre-Depends: /d' ./control |\
+         grep 'Depends: ' |\
+         sed -e 's/Depends: //g' |\
+         sed -e 's/-/_/g' |\
+         sed -e 's/+/plus/g' |\
+         sed -e 's/dpkg//g' |\
+         sed -e 's/ ([^)]*)//g' |\
+         sed -e 's/libx11/sommelier/g' |\
+         sed -e 's/libgcc1/gcc10/g' |\
+         sed -e 's/libpangocairo/pango, cairo/g' |\
+         sed -e 's/libgtk_3/gtk3/g' |\
+         sed -e 's/libglib/glib/g' |\
+         sed -e 's/libgtk_2/gtk2/g' |\
+         sed -e 's/libatspi...../at_spi2_core/g' |\
+         sed -e 's/libqt5core5a/qtbase/g' |\
+         sed -e 's/\.//g' |\
+         sed -e 's/\,//g'))
```

Comment:
I imagine this search and replace logic is not complete.

Reply:
We need more filters to fit the dependency name to `crew` one